introductions/getting-started.md

```diff
@@ -70,3 +62,34 @@ This will fetch the docker image from ghcr and fire up a container running jupyt
 - On GHA we are using our prebuild images as layer cache, but layers can't be updated on pull requests due to security reasons.
 - The erad2022 package on https://ghcr.io/openradar/erad2022:latest represents always the status of the most recent commit to the repo.
 - Running locally as well as building & running is easily possible.
+
+
+## Using the PANGEO Cloud with Binder
+
+### 1. Make Sure You Have a Github Account
+The first step is to make sure you have a Github Account.
+
+Here is the link if you do have one already:
+- [Github account creation link](https://github.com/join)
+
+### 2. Log into Pangeo Binder
+Next, sign in and authenticate the Pangeo Binder, which is the platform we will use for the workshop:
```

Comment:
I think we should mention the machine limitations here (10 GB of memory, 2 core CPU, etc.) 

And mention if they are interested in using a bigger instance, they can follow the instructions here - https://pangeo.io/cloud.html

Or at a minimum, link back to the Pangeo docs here.